test/onnx/test_fx_to_onnx.py

```diff
@@ -164,6 +164,42 @@ def forward(self, input):
             expected_error_node="aten.convolution.default",
         )
 
+    def test_dynamo_export_retains_readable_parameter_and_buffer_names(self):
+        class MNISTModel(nn.Module):
```

Comment:
How about adding a buffer? The PR handles both parameters and buffers, right?

Reply:
done